lib/src/constants/server-environments.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we remove this additional space here?

Reply:
The `lib/src/constants/server-environments.ts` file has been removed now, as we are not using environment-specific logic.